### ISSUE TITLE
Two apps different look different

### ISSUE THREAD

**Issue by zmichels** [StraboSpot/strabo-mobile#246]
If I download the app from the PhoneGap share/public page, I get an app that does not use the whole screen on my device and looks kinda weird. If I download the app trough the page link in the "about" part of the app (which links to StraboSpot), I get an app with a new icon and it fills the screen of my device like it did before.
![](http://i.imgur.com/aYp0l9M.jpg)![](http://i.imgur.com/7yS4aJm.jpg)

**Comment by jessica-azgs** [comment]
@zmichels Could you try thePhoneGap app (Strabo Mobile) again? I was reverting back to old builds all morning to try to solve the iMazing issue. Also Strabo Spot has some things not working with it yet so we're not using that app. Stick to the the old PhoneGap page to download the app. I fixed the link on the About page.

**Comment by zmichels** [comment]
Just tried from phone gap page.
Still has black bars above and below rather than fill full screen.

**Comment by zmichels** [comment]
![](http://i.imgur.com/tS1PZOR.jpg)

**Comment by jessica-azgs** [comment]
Could you test it now? I'm not getting the same thing on my iPad and I don't have an iPhone yet to test on. I think the problem happened when PhoneGap stopped supporting something with the statusbar. I changed it to the new way but there may be some other settings I have to fix with regard to that if it still doesn't look right.

**Comment by zmichels** [comment]
Sure.
Just did it again now. Still weird. Hmmmm

![](http://i.imgur.com/707BSzd.jpg)

**Comment by jessica-azgs** [comment]
What version of iOS are you using?

**Comment by zmichels** [comment]
9.3.1

**Comment by jessica-azgs** [comment]
Added <preference name="StatusBarOverlaysWebView" value="false"/>. Can you try again @zmichels? This is not a great way to test, I know, but I can't see the results of a line change on my emulator or iPad since neither has the black bars like you have.

**Comment by zmichels** [comment]
Just did again. Same thing.
I don't mind keeping trying. I'll try remember to keep checking.

**Comment by jessica-azgs** [comment]
Thanks. Just made another one. The documentation on this isn't great so feel like it's trial and error.

```
        if ($window.StatusBar) {
          // org.apache.cordova.statusbar required
          $window.StatusBar.styleDefault();
          $window.StatusBar.overlaysWebView(false);
        }

**Comment by zmichels** [comment]
NP, I'll keep trying. 

![](http://i.imgur.com/HHCjbpW.jpg)

**Comment by zmichels** [comment]
[image1.PNG]

On Apr 27, 2016, at 6:31 PM, Jessica Good <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks. Just made another one. The documentation on this isn't great so feel like it's trial and error.

        if ($window.StatusBar) {
          // org.apache.cordova.statusbar required
          $window.StatusBar.styleDefault();
          $window.StatusBar.overlaysWebView(false);
        }


-

**Comment by jessica-azgs** [comment]
Alright, going to have to look at this tomorrow and see which library, plugin, etc caused this by reverting to older versions of the app until we get to a version where this wasn't happening. Maybe I'll see if I can borrow someone's iphone here and get you the IMEI number.

I'm assuming you just noticed it today. @panamint55 is this happening on your iphone and if so when did it start?

**Comment by zmichels** [comment]
I did only notice today. I will be traveling some tomorrow... But I'll update and try as I can

**Comment by panamint55** [comment]
I have the problem with black bars as well.

Doug

***********************************************
J. Douglas Walker
Isotope Geochemistry Laboratories
Nichols Hall
2335 Irving Hill Drive
Lawrence, KS  66045
785-864-7711
785-864-7789 (fax)
***********************************************

**Comment by jessica-azgs** [comment]
Fixed with commit 4601ebd
